src/clr-angular/utils/component/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export { isAttributeSet } from './is-attribute-set';
```

Comment:
We just want to use `export * from './is-attribute-set';` format for these index files.